tools/engine_tool/lib/src/commands/run_command.dart

```diff
@@ -0,0 +1,141 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' show ProcessStartMode;
+
+import 'package:engine_build_configs/engine_build_configs.dart';
+import 'package:process_runner/process_runner.dart';
+
+import '../build_utils.dart';
+import 'command.dart';
+import 'flags.dart';
+
+/// The root 'run' command.
+final class RunCommand extends CommandBase {
+  /// Constructs the 'run' command.
+  RunCommand({
+    required super.environment,
+    required Map<String, BuilderConfig> configs,
+  }) {
+    builds = runnableBuilds(environment, configs);
+    debugCheckBuilds(builds);
+    // Add options here that are common to all queries.
```

Comment:
This comment looks like a copy paste error

Reply:
Correct. Removed.